client/overview/task-list/tasks/po-task.tsx

```diff
@@ -117,20 +120,17 @@ export const getVerifyBankAccountTask = (): any => {
 
 	return {
 		key: 'verify-bank-details-po',
-		level: level,
 		title: title,
 		content: description,
+		level: level,
 		completed: false,
-		onClick: () => {
-			window.location.href = addQueryArgs( wcpaySettings.connectUrl, {
-				collect_payout_requirements: true,
-			} );
-		},
-		actionLabel: actionLabelText,
-		visible: true,
-		expandable: true,
```

Comment:
please leave `expandable: true` here too, otherwise the redirect will happen instead of expand
this is how it works in develop
![remove-feature-flag-develop](https://github.com/Automattic/woocommerce-payments/assets/79862886/2d8e3b3b-71ec-45ac-82fc-917e834a71a1)

and this is how it currently works in the branch

![remove-feature-flag-branch](https://github.com/Automattic/woocommerce-payments/assets/79862886/4d258435-6286-4b95-a1d4-da34143bcfcc)

Reply:
Another great catch, thanks @oaratovskyi! 

Originally I made that change because of the interface for `TaskList`, I didn't see `expandable` defined in it, maybe it was a version inconsistency. I'll fix that before merge 🙂